scripts/world_bank/wdi/worldbank.py

```diff
@@ -207,8 +206,11 @@ def group_stat_vars_by_observation_properties(indicator_codes):
     """
     # All the statistical observation properties that we included.
     properties_of_stat_var_observation = ([
-        'measurementMethod', 'measurementDenominator', 'scalingFactor',
-        'sourceScalingFactor', 'unit'
+        'measurementMethod',
+        #'measurementDenominator',
```

Comment:
If we are commenting these out, wouldn't it be better to just remove/delete the lines?